<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Entry</string>
   </property>
  </widget>
  <widget class="QLabel" name="MovieTitleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Movie Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateWatchedLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Watched:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateWatchedEntry">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MovieTitleEntry">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="MovieRatingEntry">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxCount">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="MovieRatingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating (1-10):</string>
   </property>
  </widget>
  <widget class="QLabel" name="MovieGenreLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="MovieGenreEntry">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MovieLocationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Location Watched:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="MovieLocationTheaterEntry">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Theater</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="MovieLocationHomeEntry">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="MovieCommentsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="MovieCommentsEntry">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelEntryButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Entry</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
